asv_bench/benchmarks/tslibs/normalize.py

```diff
@@ -30,6 +30,10 @@ def setup(self, size, tz):
         dti = pd.date_range("2016-01-01", periods=10, tz=tz).repeat(size // 10)
         self.i8data = dti.asi8
 
+        if size == 10 ** 6 and tz is _tzs[-1]:
```

Comment:
this seems a bit fragile, so 2 recommendations:

1) let's skip tzlocal() entirely from benchmarks
2) you can define a tzlocal import that we cache (might be complicated)

Reply:
updated to use 2.

if we ever get to cython 3.0 i think i'll be able to implement a tzlocal-equivalent in cython and this may cease to be an issue

Reply:
updated + greenish